src/components/AdvancedSelect.vue

```diff
@@ -377,10 +377,10 @@ export default {
           f.push({
             header: o.label,
           });
-          f.push(...o.options.map(opt => Object.assign(opt, { parentHeader: o.label })));
+          f.push(...o.options.map(opt => Object.assign(opt, { parentHeader: o.label, selected: this.valueIsSelected(o.value) })));
         } else {
           // it's an item without group, push it to the list
-          f.push(o);
+          f.push({...o, selected: this.valueIsSelected(o.value)});
```

Comment:
Use `Object.assign` instead of destruct, because destruct it has performance issues on a large number of items.

Reply:
Done